paymentsheet/src/main/java/com/stripe/android/lpmfoundations/luxe/PaymentMethodRequirements.kt

```diff
@@ -311,6 +311,12 @@ internal val SwishRequirement = PaymentMethodRequirements(
     confirmPMFromCustomer = false,
 )
 
+internal val WeChatPayRequirement = PaymentMethodRequirements(
```

Comment:
I'm in the middle of a refactor that renders this almost useless. We still need it! But you need to add a `PaymentMethodDefinition` and add it to `PaymentMethodRegistry` as well.

Reply:
I think this is git doing something funky - the WeChatPay Requirement already exists on master - I'm adding one for Twint!